images/linux/scripts/installers/git-ftp.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+################################################################################
+##  File:  git-ftp.sh
+##  Desc:  Installs git-ftp
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+set -e
+
+echo "Install git-ftp"
+apt-get install git-ftp -y
+
+# Check installation
+git-ftp --version
```

Comment:
Why do we duplicate this step?

Reply:
I believe it can be useful to check version in image build logs

Reply:
Then maybe add some additional verbose, like "echo "Installing git-ftp: $(git-ftp --version)""?

Reply:
apt-get install git-ftp -y - already provides details with package version

Reply:
Yes, you're right, removed it.